.github/SECURITY.md

```diff
@@ -0,0 +1,32 @@
+## Reporting Vulnerabilities
+
+We’re extremely grateful for security researchers and users that report vulnerabilities to the Hasura GraphQL Community. All reports are thoroughly investigated by a set of community volunteers.
+
+To make a report, please email us at [build@hasura.io](mailto:build@hasura.io) with the complete security details attaching all necessary information.
+
+### When Should I Report a Vulnerability?
+
+- You think you discovered a potential security vulnerability in the Hasura GraphQL Engine or related components.
+- You are unsure how a vulnerability affects the Hasura GraphQL Engine.
+- You think you discovered a vulnerability in another project that Hasura GraphQL Engine depends on (e.g. heroku, docker, etc)
+- Any other risk that would harm the Hasura GraphQL Engine.
+
+### When Should I NOT Report a Vulnerability?
+
+- You need help tuning Hasura GraphQL Engine components for security.
+- You need help applying security related updates.
+- Your issue is not security related.
+
+## Security Vulnerability Response
+
+Each report is acknowledged and analyzed by Product Security Team members within 3 working days. 
+
+As the security issue moves from triage, to identified fix, to release planning we will keep the reporter updated.
+
+## Public Disclosure Timing
+
+A public disclosure date is negotiated by the Hasura GraphQL product security team and the bug submitter. We prefer to fully disclose the bug as soon as possible once a user mitigation is available. It is reasonable to delay disclosure when the bug or the fix is not yet fully understood, the solution is not well-tested, or for vendor coordination. The timeframe for disclosure is from immediate (especially if it's already publicly known) to a few weeks. As a basic default, we expect report date to disclosure date to be on the order of 7 days. The Hasura GraphQL product security team holds the final say when setting a disclosure date.
```

Comment:
1) Change`As a basic default, we expect report date to disclosure date to be on the order of 7 days.` to `We expect the time-frame between a report to a public disclosure to typically be in the order of 7 days.`
2) Change `The Hasura GraphQL product security team holds the final say when setting a disclosure date.` -> `The Hasura GraphQL Engine maintainers and the security team will take the final call on setting a disclosure date.`